beast/physicsmodel/stars/isochrone.py

```diff
@@ -617,7 +617,7 @@ def _clean_cols(self, iso_table):
         iso_table.add_column('logg', iso_table['log_g'][:])
         iso_table.add_column('stage', iso_table['phase'][:])
         iso_table.remove_columns(['log10_isochrone_age_yr', 'log_Teff', 'log_L', 'log_g',
-                                  'initial_mass', 'star_mass', 'stage'])
+                                  'initial_mass', 'star_mass', 'phase'])
```

Comment:
Does this change the outputs? There is a failure in the travis testing. What is the difference between 'stage' and 'phase'?

Reply:
No: this is simply a correction in the column name.  The original column name is 'phase' which is copied+renamed to stage in line 619.  The incorrect column was being deleted to clean up the data file - this is now corrected.